### PR TITLE
gogo protobuf support

### PR DESCRIPTION
**Pull request by chai-nadig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Register a JSON codec with the gRPC server. In Go, it can be automatically regis
 
 `import _"github.com/jnewmano/grpc-json-proxy/codec"`
 
+If you're using `gogo/protobuf` as your protobuf backend, import the following:
+
+`import _"github.com/jnewmano/grpc-json-proxy/gogoprotobuf/codec"`
```
Comment (jnewmano) [inline README.md]: Thank you for the PR. In order to facilitate examples and support for other languages and implementations, what do you think the folder/package structure could be like? 

Perhaps renaming `gogoprotobuf/codec` to `codec/gogoprotobufcodec` would be better. 

The original codec could be aliased to something like `codec/protobufcodec`? 

This still makes it Go centric.

Comment (chai-nadig) [reply README.md]: Absolutely. If I move the `gogoprotobuf` codec under `codec/gogoprotubuf`, wouldn't both codecs initialize for folks using the older version of `grpc-json-proxy` ?

**Comment by chai-nadig** [comment]
@jnewmano would like to merge this. had a question for you about the package structuring.

**Comment by jnewmano** [comment]
I'll merge it as is. If necessary, reorganizing the codecs in a more structured package structure can be done.

**Pull request merged**